base_classes/NXapm_charge_state_analysis.nxdl.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<!--
+# NeXus - Neutron and X-ray Common Data Format
+#
+# Copyright (C) 2024-2025 NeXus International Advisory Committee (NIAC)
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXapm_charge_state_analysis" extends="NXprocess" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <symbols>
+        <doc>
+            The symbols used in the schema to specify e.g. dimensions of arrays.
+        </doc>
+        <symbol name="n_cand">
+            <doc>
+                The number of (molecular) ion candidates.
+            </doc>
+        </symbol>
+        <symbol name="n_ivec_max">
+            <doc>
+                Maximum number of allowed atoms per (molecular) ion (fragment).
+            </doc>
+        </symbol>
+        <symbol name="n_variable">
+            <doc>
+                Number of entries
+            </doc>
+        </symbol>
+    </symbols>
+    <doc>
+        Base class to document the parameters (configuration) and results of a processing for recovering
```

Comment:
[non-blocking]

Again, the use of parentheses makes this text harder to understand.  Please consider rephrasing the text to remove the parentheses.